integration_tests/.scripts/integration_test.sh

```diff
@@ -23,38 +23,44 @@ fi
 
 for db in ${DATABASES[@]}; do
 
-  echo "Snowplow unified integration tests: Running without seeding data"
-  dbt run --full-refresh --target $db
+  # echo "Snowplow unified integration tests: Running without seeding data"
+  # dbt run --select snowplow_unified_base_sessions_lifecycle_manifest --target $db
 
-  status=$?
+  # status=$?
   
-  if [ $status -ne 0 ] && [ $status -ne 1 ]; then
-    echo "dbt command failed for target $db with exit status $status"
-    exit $status
+  # if [ $status -ne 0 ] && [ $status -ne 1 ]; then
+  #   echo "dbt command failed for target $db with exit status $status"
+  #   exit $status
     
-  fi
+  # fi
 
   echo "Snowplow unified integration tests: Seeding data"
-  eval "dbt seed --full-refresh --target $db" || exit 1;
+  eval "dbt seed --full-refresh --target $db --no-partial-parse" || exit 1;
```

Comment:
Didn't we see that the --no-partial-parse did not work as a solution? Should we remove them not to clutter things?

Reply:
Will remove them!